docs/installation/install-adaptdl.rst

```diff
@@ -19,7 +19,7 @@ The AdaptDL scheduler can be installed in just one command using Helm:
 
 .. code-block::
 
-   $ helm install adaptdl adaptdl --repo https://github.com/petuum/adaptdl/raw/helm-repo \
+   $ helm install adaptdl adaptdl-sched --repo https://github.com/petuum/adaptdl/raw/helm-repo \
```

Comment:
With this change, we will need to clean up the helm branch, otherwise `helm install adaptdl adaptdl --repo https://github.com/petuum/adaptdl/raw/helm-repo` will still work, but install a stale version.

Reply:
Yup planning to do that manually